tasks/internal/get-advisory-severity/get-advisory-severity.yaml

```
@@ -119,6 +120,100 @@ spec:
               fi
           }
 
+          # Function to process a batch of CVEs with its own token
+          process_cve_batch() { # Expected arguments are [batch_id, batch_cves]
+              echo "Batch $1: Getting token..."
+              set +x
+              batch_token=$(curl --retry 3 --negotiate -u : "${OSIDB_URL}"/auth/token | jq -r '.access')
+              set -x
```

Token fetch failure is not checked
```suggestion
              set -x
              # Validate token was obtained successfully
              if [[ -z "$batch_token" || "$batch_token" == "null" ]]; then
                  echo "Batch $1: ERROR - Failed to obtain authentication token" >&2
                  return 1
              fi
```

---

Not new to this commit, but this is a great catch. Fixed